### DISCUSSION TITLE
Home Assistant + Teslamate

### DISCUSSION THREAD

**Discussion by Tiz-B** [indomus/forum#2007] [GENERALE - Discussioni generiche]
Salve,
ho installato Teslamate su di un Pi4 e di recente ho anche installato HA e volevo far interagire le due cose: ```https://docs.teslamate.org/docs/integrations/home_assistant/```

Ho provato a cercare in giro come fare ma non capisco come e dove inserire gli script yaml - sarà che l'ultima versione ha cambiato un po' le cose, tipo non trovo lo store... di certo ho appena iniziato e ci capisco ben poco...

Grazie

**Comment by snakuzzo** [comment]
Guarda che è scritto tutto passo passo al link che hai postato. Poi non capisco di che store parli.
Se non ti è chiaro cosa fare, ti consiglio di leggere un po' di guide partendo dalle basi. Sul sito noi abbiamo un percorso iniziale che potrebbe chiarirti tante cose

**Reply by Tiz-B** [reply]
Come dicevo, ci capisco ben poco - mi riferivo alla raccolta di componenti che non ho idea di dove si trovi.
Ho visionato varie guide ma si riferiscono alla versione precedente di HA - molte cose sono cambiate.

**Reply by snakuzzo** [reply]
Non ti seguo. Che c'entra la versione vecchia e nuova?
Di che versione parli?
Che tipo di installazione hai?

**Reply by snakuzzo** [reply]
Se ci capisci poco, ripeto, segui prima le guide base...non partire a cannone con il voler fare tutto subito, perché sbagli

**Reply by Tiz-B** [reply]
Finalmente ho capito... ho installato la versione core su docker-compose... devo installare la versione completa...

**Reply by snakuzzo** [reply]
Versione completa non vuol dire nulla. 
Devi sempre considerare che gli addon di HA OS sono dei container docker. Quindi quello che fa l'addon puoi farlo anche a parte come altro container docker.
